### PR TITLE
Integrate test report for ginkgo test and cucushift test

### PR DESCRIPTION
**Pull request by liangxia**
If any of the tests in openshift-tests-private or verification-tests/cucushift failed, we signal that in the report step ( as the last step in tests).

### CODE REVIEW

**Comment by liangxia** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.11-amd64-nightly-aws-ipi-ovn-ipsec-f28 periodic-ci-openshift-openshift-tests-private-release-4.10-amd64-nightly-aws-ipi-minimal-permission-f28

**Comment by liangxia** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.11-amd64-nightly-aws-ipi-ovn-ingress-nlb-f28 periodic-ci-openshift-openshift-tests-private-release-4.11-amd64-nightly-azure-mag-ipi-fullyprivate-f28 periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-aws-ipi-ovn-hypershift-mgmt-critical-f7

**Comment by liangxia** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.11-amd64-nightly-aws-ipi-ovn-ingress-nlb-f28 periodic-ci-openshift-openshift-tests-private-release-4.11-amd64-nightly-azure-mag-ipi-fullyprivate-f28 periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-aws-ipi-ovn-hypershift-mgmt-critical-f7

**Comment by liangxia** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.11-amd64-nightly-aws-ipi-ovn-ingress-nlb-f28 periodic-ci-openshift-openshift-tests-private-release-4.11-amd64-nightly-azure-mag-ipi-fullyprivate-f28 periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-aws-ipi-ovn-hypershift-mgmt-critical-f7

**Comment by liangxia** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.11-amd64-nightly-aws-ipi-ovn-ingress-nlb-f28 periodic-ci-openshift-openshift-tests-private-release-4.11-amd64-nightly-azure-mag-ipi-fullyprivate-f28 periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-aws-ipi-ovn-hypershift-mgmt-critical-f7

**Comment by liangxia** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.11-amd64-nightly-aws-ipi-ovn-ingress-nlb-f28 periodic-ci-openshift-openshift-tests-private-release-4.11-amd64-nightly-azure-mag-ipi-fullyprivate-f28 periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-aws-ipi-ovn-hypershift-mgmt-critical-f7

**Comment by liangxia** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.11-amd64-nightly-aws-ipi-ovn-ipsec-f28 periodic-ci-openshift-openshift-tests-private-release-4.10-amd64-nightly-aws-ipi-minimal-permission-f28

**Comment by liangxia** [comment]
/pj-rehearse  periodic-ci-openshift-openshift-tests-private-release-4.10-amd64-nightly-aws-ipi-minimal-permission-f28

**Comment by liangxia** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.10-amd64-nightly-aws-ipi-minimal-permission-f28 periodic-ci-openshift-openshift-tests-private-release-4.11-amd64-nightly-aws-ipi-ovn-ipsec-f28 periodic-ci-openshift-openshift-tests-private-release-4.11-amd64-nightly-aws-ipi-ovn-ingress-nlb-f28

**Comment by liangxia** [comment]
Verified this is working as expected,
```
INFO[2023-09-18T01:52:20Z] Running step aws-ipi-ovn-ipsec-f28-openshift-extended-test. 
INFO[2023-09-18T03:25:31Z] Step aws-ipi-ovn-ipsec-f28-openshift-extended-test succeeded after 1h33m11s. 
INFO[2023-09-18T03:25:31Z] Running step aws-ipi-ovn-ipsec-f28-cucushift-e2e. 
INFO[2023-09-18T04:45:14Z] Step aws-ipi-ovn-ipsec-f28-cucushift-e2e succeeded after 1h19m42s. 
INFO[2023-09-18T04:45:14Z] Running step aws-ipi-ovn-ipsec-f28-openshift-extended-web-tests. 
INFO[2023-09-18T05:39:09Z] Step aws-ipi-ovn-ipsec-f28-openshift-extended-web-tests succeeded after 53m55s. 
INFO[2023-09-18T05:39:09Z] Running step aws-ipi-ovn-ipsec-f28-openshift-e2e-test-qe-report. 
INFO[2023-09-18T05:39:16Z] Logs for container test in pod aws-ipi-ovn-ipsec-f28-openshift-e2e-test-qe-report: 
INFO[2023-09-18T05:39:16Z] 5 failures in openshift-extended-test
Please investigate these ginkgo test failures from build artifacts
39 failures in cucushift-e2e
Please investigate these cucushift failures from build artifacts
```

/cc @jhou1 @kuiwang02 @heliubj18 @anpingli

File: ci-operator/step-registry/openshift-extended/test/openshift-extended-test-commands.sh [hunk 1]
```diff
@@ -293,17 +293,29 @@ function run {
...
     # please refer to the junit result for case result, not depends on step result.
     if [ "W${FORCE_SUCCESS_EXIT}W" == "WnoW" ]; then
         echo "force success exit"
         exit 1
     fi
-    echo "normal exit"
-    exit 0
+
+    # summarize test results
+    echo "Summarizing test result..."
```
Comment (kuiwang02) [inline ci-operator/step-registry/openshift-extended/test/openshift-extended-test-commands.sh:307]: @liangxia 
the following steps also need it?
openshift-extended-test-longduration
openshift-extended-test-disruptive
openshift-extended-logging-test

I saw the chain you add openshift-e2e-test-qe-report into includes the above step.

Comment (liangxia) [reply ci-operator/step-registry/openshift-extended/test/openshift-extended-test-commands.sh:313]: Added.

**Comment by heliubj18** [comment]
LGTM

**Pull request merged**